asset/js/setup.js

```diff
@@ -37,6 +43,7 @@ function setup() {
   const vm = new Vue({
     el: '#app',
     mounted() {
+      flatNestedModals();
       scrollToUrlAnchorHeading();
```

Comment:
I wonder if it would be a good idea to extract out a method here, so that future developers don't have to modify two different places whenever he or she wants to add a new routine:

```javascript
function executeAfterMountedRoutines() {
  flattenModals();
  scrollToUrlAnchorHeading();
}
```